src/internal/query.ts

```diff
@@ -17,10 +17,14 @@ type KeyOfObject<T> = Extract<keyof T, string>;
  * <T> - generic resource object
  */
 export interface IAggField<T = any> {
-  fn: "COUNT" | "MIN" | "MAX" | "AVG" | "SUM" | "EVERY";
+  fn: "count" | "min" | "max" | "avg" | "sum";
```

Comment:
what do you think about providing this as an `enum` or even provide some functions for each that could be used like 
``` js
posts.select().fields("author", max("likes"))

Reply:
I think very positively about this and I also have a plan to simplify the SDK (there is a card about this), where I'm going to introduce `jexia.min()`, `jexia.max()` and others. These function will return the same `IAggField` object, so this code will not be changed